web_src/js/modules/tippy.ts

```diff
@@ -209,3 +209,7 @@ export function showTemporaryTooltip(target: Element, content: Content): void {
     },
   });
 }
+
+export function getAttachedTippyInstance(el: Element): Instance | null {
+  return el._tippy ?? null;
+}
```

Comment:
Instead of  this, can you adjust `globals.d.ts` to add `undefined` to the property:

```ts
interface Element {
  _tippy: import('tippy.js').Instance | undefined;
}
```

And then just use `el._tippy`, typescript will then take care that the undefined case is handled.